TP2a0.10/expt_01.0/blkdat.input

```diff
@@ -205,17 +205,19 @@ S-Z(15-11): dp00/f/x/i=3m/1.125/12m/1m; ds=1m/1.125/4m; src_2.2.12;
   -1	  'lwflag' = longwave (SST) flag (0=none,1=clim,2=atmos)
    0	  'sstflg' = SST relaxation flag (0=none,1=clim,2=atmos,3=observed)
    0	  'icmflg' = ice mask       flag (0=none,1=clim,2=atmos,3=obs/coupled)
-1.e5      'prsbas' = msl pressure is input field + prsbas (Pa)
```

Comment:
I know that 1 Pa is not much but I think the real value is 0 when MSLP is in absolute value.